lib/puppet/provider/iis_application_pool/webadministration.rb

```diff
@@ -12,39 +12,47 @@
   mk_resource_methods
   
   def create
-    inst_cmd = "New-WebAppPool -Name \"#{@resource[:name]}\" -ErrorAction Stop"
-    result   = self.class.run(inst_cmd)
+    cmd = []
```

Comment:
Just meant it wasn't relevant to this PR and made it hard to determine what changed. Nitpick.